### DISCUSSION TITLE
New Model in Livewire Component

### DISCUSSION THREAD

**Discussion by alex1897** [livewire/livewire#5178] [1. Help] [answered]
I am not sure if this is a bug of Livewire or if I am just missing knowledge. How can I add to a public Livewire component variable (array) with a public function from the Livewire view, a new Model?

That is my current setup:
<img width="602" alt="Bildschirmfoto 2022-08-13 um 11 03 08" src="https://user-images.githubusercontent.com/6780121/184477208-6a37596f-38d6-4ef8-9b7b-05c13eaa28d6.png">

While the dd() in the mount method brings the expected result:
<img width="315" alt="Bildschirmfoto 2022-08-13 um 11 05 47" src="https://user-images.githubusercontent.com/6780121/184477224-5ea9236e-56aa-4dfa-a748-5aafd9631e16.png">

Does my add() method mess up my array:
<img width="326" alt="Bildschirmfoto 2022-08-13 um 11 06 07" src="https://user-images.githubusercontent.com/6780121/184477236-8529feb7-0b25-477d-a68d-c39afeee8088.png">

**Comment by ju5t** [comment, marked as answer]
It's better to work with arrays. And if you can't, you need to make sure `QuestionData` can be hydrated and dehydrated. 

See: https://laravel-livewire.com/docs/2.x/properties#wireable-properties.

**Comment by alex1897** [comment]
@ju5t: Yes, that is what I already did and works. Thanks!
